install/kubernetes/helm/istio/templates/tests/test-citadel-connection.yaml

```diff
@@ -0,0 +1,30 @@
+{{- if .Values.global.enabledTest }}
+apiVersion: v1
```

Comment:
citadel was made an optional component.  I'm not sure why, but security.enabled is the flag.

Reply:
done